src/google/protobuf/io/io_win32.cc

```diff
@@ -0,0 +1,281 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
Please move this to src/google/protobuf/stubs/.... That's where we put all base (and platform related) dependencies.